.cursor/scripts/common_util.py

```
@@ -0,0 +1,251 @@
+            dest_files = list(Path(destination).rglob("*"))
+            result["file_count"] = len(dest_files)
+            
+            if len(source_files) != len(dest_files):
+                result["error"] = f"파일 개수 불일치: 원본 {len(source_files)}, 대상 {len(dest_files)}"
+                return result
+        else:
+            # 빠른 카운트 (최상위만)
+            result["file_count"] = len(os.listdir(destination))
+        
+        result["success"] = True
+        return result
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)